aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -545,19 +549,19 @@ module aptos_framework::account {
         alice
     }
 
-/*
-TODO bring back with generic rotation capability
```

Comment:
Let's just kill this TODO. Whoever wants to add this back in the future can go through the history and find it.